chord_metadata_service/chord/workflows/wdls/cbioportal_export.wdl

```diff
@@ -98,14 +110,17 @@ task get_maf {
             no_file_processed_yet = True
             for i, maf_uri in enumerate(file_handle):
 
-                # Request from DRS the maf file absolute local path
+                # Request the MAF file from DRS
                 object_id = maf_uri.split("/")[-1].rstrip()
-                drs_object_url = f"~{drs_url}/objects/{object_id}?internal_path=1"
-                response = requests.get(drs_object_url, verify=False)
+                response = requests.get(
+                    f"~{drs_url}/objects/{object_id}?internal_path=1",
+                    headers={"Authorization": "Bearer ~{token}"},
+                    verify=~{true="True" false="False" validate_ssl},
```

Comment:
String literals are only for cert paths for the `verify` arg

> verify – (optional) Either a boolean, in which case it controls whether we verify the server’s TLS certificate, or a string, in which case it must be a path to a CA bundle to use. Defaults to True.

Reply:
It may be fine, from WDL [docs](https://github.com/openwdl/wdl/blob/main/versions/1.0/SPEC.md#expression-placeholder-options):
> Expression placeholder options are option="value" pairs that precede the expression in an expression command part and **customize the interpolation of the WDL value into the command string being built.**